test_runner/src/main/kotlin/ftl/args/AndroidArgs.kt

```diff
@@ -13,6 +13,7 @@ data class AndroidArgs(
     val roboScript: String?,
     val environmentVariables: Map<String, String>, // should not be printed, becuase could contains sensitive informations
     val directoriesToPull: List<String>,
+    val grantPermissions: String?,
```

Comment:
I guess that this property should be used somewhere, but it is not or I cannot find.

Reply:
it will be used below. with @pawelpasterz comment